test/server/cards/01_SOR/upgrades/SmugglingCompartment.spec.ts

```diff
@@ -0,0 +1,51 @@
+describe('Smuggling Compartment', function() {
+    integration(function(contextRef) {
+        describe('Smuggling Compartment\'s on attack ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'atst', upgrades: ['smuggling-compartment'] }, { card: 'wampa', upgrades: ['smuggling-compartment'] }],
```

Comment:
why do you have wampa with this upgrades ?

Reply:
This was just left over bad code. Thanks!